Example/Example.xcodeproj/project.pbxproj

```diff
@@ -333,7 +375,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 4.2;
```

Comment:
I updated #67 since it has done good manual work as well. Can you please remove update from PR to 4.2 in order to solve merge conflicts?